documentation/modules/exploit/linux/fileformat/debian_package_hijacker.md

```diff
@@ -0,0 +1,161 @@
+## Vulnerable Application
+
+This module exploits the widespread lack of per-package signing
+in the Debian package ecosystem. Most `.deb` files are vulnerable
+to manipulation if read-write access is available. By injecting a
+payload into the package we can gain execution as the root user.
+Custom dependencies can also be specified, allowing us to install
+additional software. This module can be run with a handler, but it
+is not enabled by default.
+
+The exploit module was authored by:
+
+- Andrew Vanderbilt @ Vander Security (Original PoC)
+- Nick Cottrell <Rad10Logic> (PoC Additions)
+
+## Verification Steps
+
+ 1. Start msfconsole
+ 2. Get a root session
+ 3. `use exploit/linux/fileformat/debian_package_hijacker`
+ 4. `set SESSION <id>`
+ 5. `set PACKAGE <package_name>` # should be either path to local package or name of apt package
+ 6. `set DEPENDENCIES <extra_dependencies>`
+ 7. `set INSTALL_PACKAGE true`
+ 8. `set DisablePayloadHandler false`
+ 9. `set ExecuteOnInstall true`
+10. `exploit -z`
+11. Ensure you have a 2nd root session
+
+
+## Options
+
+The module provides several configurable options:
+
+- `PACKAGE` (required): The absolute file path to the Debian package
+to hijack.
```

Comment:
Can we update this to the markdown format as shown here: https://github.com/rapid7/metasploit-framework/blob/b4991a97d02572a323470aa0906e934cce1b7843/documentation/modules/module_doc_template.md#L20-L25